playbooks/firmware.yml

```diff
@@ -0,0 +1,6 @@
+- name: Update DPU Firmware
+  hosts: all
+  connection: local
```

Comment:
This can be moved to the task level so that user has no control on where the tasks runs.

Reply:
done